aur.py

```diff
@@ -15,16 +16,32 @@ def package_installed(module, package_name):
     return exit_code == 0
 
 
-def install_packages(module, package_name, tool):
+def update_packages(module, tool, auronly):
+    assert tool in TOOL_CMD_MAP
+
+    cmd = ['env', 'LC_ALL=C'] + TOOL_CMD_MAP[tool] + ['-Su']
+    if auronly:
+        cmd += ['--aur']
+    rc, stdout, stderr = module.run_command(cmd, check_rc=True)
+
+    module.exit_json(
+        changed='there is nothing to do' not in stdout,
+        msg='updated packages',
+    )
+
+
+def install_packages(module, package_name, tool, auronly):
     if package_installed(module, package_name):
         module.exit_json(
             changed=False,
             msg='package already installed',
         )
 
-    assert tool in TOOL_TO_INSTALL_CMD_MAP
+    assert tool in TOOL_CMD_MAP
 
-    cmd = TOOL_TO_INSTALL_CMD_MAP[tool] + [package_name]
+    cmd = TOOL_CMD_MAP[tool] + ['-S', package_name]
+    if auronly:
```

Comment:
This is implied by calling the `aur` handler.

Reply:
What do you recommend ? That the auronly option is set to true by default ?

Reply:
Yeah, it should be the default.

Reply:
ok if I change auronly to default True do you think my proposal is acceptable?